user_tools/docs/user-tools-onprem.md

```diff
@@ -238,25 +238,25 @@ The local deployment runs on the local development machine. It requires:
 
 | Option               | Description                                                                                                                                                                                                                                               | Default                                                                                                                                       | Required |
 |----------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|----------|
-| **worker_info**      | A path pointing to a yaml file containing the system information of a worker node. It is assumed that all workers are homogenous. The format of the file is described in the following section.                                                           | None                                                                                                                                          | Y        |
+| **executor_info**      | A path pointing to a yaml file containing the system information of a executor node. It is assumed that all executors are homogenous. The format of the file is described in the following section.                                                           | None                                                                                                                                          | Y        |
```

Comment:
This could be out of alignment with the table.

Reply:
Fixed